doc/design/volumes/ContainerCreateWithVolumes.md

```diff
@@ -1,34 +1,36 @@
 
-# Container Create w/ Volumes
+### Docker Personality
 
+The docker personality needed no changes to support nfs volumes. This was due to our Volume Store architecture. Adding a new type of Volume Store still appears as a volumestore to the docker personality. In the future we are planning to distinguish between volume types by customizing our `driver` field. e.g. `vsphere-vmdk` and `vsphere-nfs`. 
 
-### Docker Personality
+This will change the way the personality operates, changes will need to be made on how we identify the volume store in order to know which kind of driver we need to return on ls. Driver can also be specified at volume creation time and it is very possible that we will require users to know which volume store is of which driver type. In order to facilitate this we should modify the output of `docker info` to list not only the available volume stores, but also the drivers associated with each store. Alternatively/additionally we can coax which type of driver belongs to a store programatically, this will require some work in the portlayer and docker around fetching a store by name and determining it's type. 
 
-Changes to this portion will involve interacting with the new planned changes to the portlayer.
+In the future some mount options like `squashfs`, `sync`, and `retries` may be desired as configuration options at volume creation time. If this is the case work will need to be done on the personality to parse the additional `driver arguments`. Naturally, the portlayer will also need some additional argument parsing logic to facilitate communicating these options to the tether at mount/run time. 
 
-    It will involve determining whether a volume exists or needs to be created and then asking the portlayer to perform a join operationon the obtained volume. The changes here will mainly be looking for the new inputs to the `docker create` command that we want to support. This call is really managing and informing a call to the VolumeCreate call in the case of anonymous volume creation, if the volume already exists then a previous `VolumeCreate` can be assumed and only a joing will be necessary after fetching the existing volume's data. 
+This will also require us to keep a basic set of options for anonymous volumes. At create time users can actually specify several options after the ending `:` character, some investigation will be needed as to how we can use this. The basic options can be used when nothing is specified at the end of the `-v` option on create. We also have a hard coded default capacity for anonymous volumes as this cannot be specified from the docker cli. In the future we should make this configurable at vic-machine create time, [Ticket Reference](https://github.com/vmware/vic/issues/5172). 
 
 examples:
 
 ```
+
 docker create -v "/mnt/pnt:<some basic opts>"
 
 and 
 
 docker create -v "name:/mnt/pnt:<some basic opts>"
```

Comment:
Might want to format this as:  `docker create -v "<name>:/mnt/pnt:<some basic opts>"`, if that's what you were going for, otherwise ignore.